src/c-writer.cc

```diff
@@ -1527,7 +1603,7 @@ void CWriter::WriteFuncDeclarations() {
   for (const Func* func : module_->funcs) {
     bool is_import = func_index < module_->num_func_imports;
     if (!is_import) {
-      Write("static ");
+      Write(InternalSymbolDeclScope());
```

Comment:
Because this is the only place `InternalSymbolDeclScope` gets called, and because I think `extern` is the default for function declarations, I *think* this could just be a call to `InternalSymbolScope` and we could eliminate the `InternalSymbolDeclScope` method entirely.